src/mgr/BaseMgrModule.cc

```diff
@@ -125,12 +125,20 @@ ceph_send_command(BaseMgrModule *self, PyObject *args)
 
   char *cmd_json = nullptr;
   char *tag = nullptr;
+  char *inbuf_ptr = nullptr;
+  Py_ssize_t inbuf_len = 0;
+  bufferlist inbuf = {};
+
   PyObject *completion = nullptr;
-  if (!PyArg_ParseTuple(args, "Ossss:ceph_send_command",
-        &completion, &type, &name, &cmd_json, &tag)) {
+  if (!PyArg_ParseTuple(args, "Ossssz#:ceph_send_command",
```

Comment:
It'd be helpful to have these changes in a separate commit, explaining the additon of inbuf, for future reference. Let's run this PR through a rados run, if it hasn't been run already.

Reply:
@neha-ojha: that *was* in a separate commit... but now it seems that all commits in this PR have been squashed :-/

Reply:
> @neha-ojha: that _was_ in a separate commit... but now it seems that all commits in this PR have been squashed :-/
When I previously warned (it was in the PR description) that I was about to squash commits there were no objections from anyone so I did it.
Nevertheless I'll create a separate commit for this change.